lib/commons/color/color.js

```diff
@@ -57,28 +129,12 @@ export default class Color {
    * @instance
    */
   parseString(colorString) {
-    // Colorjs currently does not support rad or turn angle values
```

Comment:
[Colorjs 0.5.0](https://github.com/color-js/color.js/releases/tag/v0.5.0) now fully supports grad, rad, and turn so we don't need to do this anymore.